mod/caching/set/type_plus_right/wikirate_analysis/metric.rb

```diff
@@ -0,0 +1,33 @@
+include Card::CachedCount
+
+expired_cached_count_cards do |changed_card|
+  # "right_plus":[["topic",{"refer_to":"_2"}],
+  #  ["_ll",{"right_plus":["*cached count",{"ne":0}]}] ]
+  expired_cached_cards = []
+  if (r = changed_card.right)
+    # metric's topic changes
+    if (l = changed_card.left) && l.type_code == :metric &&
+       r.key == 'topic' && changed_card.type_code == :pointer
+      analysis_type_id = Card::WikirateAnalysisID
+      # find all related analysis to the topic
+      card_names = changed_card.item_names.unshift('in')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.